CONTRIBUTING.md

```diff
@@ -84,7 +84,23 @@ Before and/or after code review, clean up your commit history. If the `main` bra
+Once the pull request has been approved, we expect _you_ to merge. Although this pulls your commits into the `main` branch, it does not yet deploy your changes to the `main` production pipeline. RECOVER data has FISMA restrictions, but only our production account is FISMA compliant. Since there is no guarantee that the test data provided to us (which doesn't have FISMA restrictions) perfectly models the production dataset, we maintain a `staging` namespace in the production account which enables us to test changes on production data before pulling those changes into the `main` namespace. Merging into `main` will deploy the changes to the `staging` namespace. To complete deployment to the `main` namespace of production, we push a new tag with a specific format to this repository, which will trigger [this GitHub action](.github/workflows/upload-and-deploy-to-prod-main.yaml). There is a diagram of this process [here](https://sagebionetworks.jira.com/wiki/spaces/IBC/pages/2863202319/ETL-390+RECOVER+Integration+Testing#Implementation). See [release process](#release-process) for more details on this.
+
+### Release process
+
+This package uses [semantic versioning](https://semver.org/) for releasing new versions.
+
+**Background Pages:**
+
+- See [About releases - GitHub Docs](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) for more information about Github releases and their relationship to github tags
+- See [Git - Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) for more information about Github tags
+
+**Instructions:**
+
+1. Once there are important features that need to be added so it’s applied to the production data, we will need to create a new tag and release
+1. [Create a new tag and release on the repo](https://github.com/Sage-Bionetworks/recover/releases) through here on the `main` branch. This will auto-create the new tag that you specify and release notes. Also include any known bugs for each release here.
```

Comment:
```suggestion
1. [Draft a new release for the repo here](https://github.com/Sage-Bionetworks/recover/releases), specify the `main` branch as the target, and choose a tag name. Click `Generate release notes`, review the content and be sure to include any known bugs for the release.
```

Reply:
Note: may want to add that you can specify a tag name that doesn't exist to leverage the github auto tag create feature.